src/middlewares/validation.js

```diff
@@ -0,0 +1,30 @@
+import Joi from '@hapi/joi';
+import Schemas from '../utils/schemas';
+
+const validation = (req, res, next) => {
+  const methodsSupported = ['post', 'put', 'patch'];
+  const { path } = req.route;
+  const method = req.method.toLowerCase();
+
+  if (methodsSupported.includes(method) && Schemas[path] !== undefined) {
+    const schema = Schemas[path];
+
+    return Joi.validate(req.body, schema, (error, data) => {
+      if (error) {
+        res.status(400).json({
+          status: 400,
+          error: error.details[0].message,
+        });
+      }
+
+      req.body = data;
+      next();
+    });
+  }
+  res.status(405).json({
```

Comment:
You can use the Response handler for this #18

Reply:
Done now